src/components/APIDoc/Operation.tsx

```diff
@@ -46,6 +50,20 @@ export const Operation: React.FunctionComponent<OperationProps> = props => {
 const OperationContent: React.FunctionComponent<OperationProps> = ({verb, path, operation, document}) => {
   const parameters = (operation.parameters || []).map(p => deRef(p, document));
 
+  const reqData: RequestFormat = {
+    method: verb.toUpperCase(),
+    url: "http://example.com"+path,
+    httpVersion: "HTTP/1.1",
+    cookies: [],
+    headers: [{name: "Accept", value: "application/json"}],
+    queryString: [], //TODO path params?
+    postData: undefined, //TODO body params
+    headersSize: -1,
+    bodySize: -1,
+  }
```

Comment:
Lets memoize this one:

```suggestion
  const reqData: RequestFormat = useMemo(() => ({
    method: verb.toUpperCase(),
    url: "http://example.com"+path,
    httpVersion: "HTTP/1.1",
    cookies: [],
    headers: [{name: "Accept", value: "application/json"}],
    queryString: [], //TODO path params?
    postData: undefined, //TODO body params
    headersSize: -1,
    bodySize: -1,
  }), [verb, path]);
```